tools/templates/wezterm.toml

```diff
@@ -0,0 +1,19 @@
+# {{ scheme_name }}
+[colors]
+foreground = "#{{ Foreground_Color.hex }}"
+background = "#{{ Background_Color.hex }}"
+{% if Cursor_Color %}
+cursor_bg = "#{{ Cursor_Color.hex }}"
```

Comment:
Please also add:

```
cursor_border = "#{{ Cursor_Color.hex }}"
```

to match #261

Reply:
Thanks for the review. Added cursor_border to template.